etcdserver/server.go

```diff
@@ -928,6 +933,33 @@ func (s *EtcdServer) applyConfChange(cc raftpb.ConfChange, confState *raftpb.Con
 	return false, nil
 }
 
+func (s *EtcdServer) createRaftSnapshot(snapi uint64, confState raftpb.ConfState) {
```

Comment:
add a comment to this method please. what is the difference between createRaftSnapshot and snapshot?